src/components/tools/jobs.js

```
@@ -0,0 +1,68 @@
+    list.forEach(j => {
+        const delay = (
+            (j.finishedOn ? j.finishedOn : Math.floor(Date.now()))
+            - j.processedOn
+        ) / 1000;
+        const progress = j.status && j.status === "completed" ?
+            "100" : j.progress && j.progress.percentage ?
+                Math.round(j.progress.percentage) : 0
+        tmpJobs.push({
+            rows: j.data.totalRows,
+            id: j.id,
+            user: j.data && j.data.user,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)